ext/mysqli/tests/mysqli_ssl_set.phpt

```diff
@@ -4,13 +4,15 @@ mysqli_ssl_set() - test is a stub!
 mysqli
 --SKIPIF--
 <?php
-require_once('skipifconnectfailure.inc');
+die("skip test is a stub!");
```

Comment:
Not sure what to do with this test. It is apparently "a stub" since its introduction to the code base 234 years ago.